_pyproject.toml

```diff
@@ -0,0 +1,5 @@
+# Specify build system requirements for pip wheel builds.
```

Comment:
I think this is pretty confusing for a user. Maybe hide it inside the .github/workflows directory instead?  or does that still mess up setuptools?

Reply:
another option is to ditch setup.py, but I think it still might be hard to customize a C module build in pyproject

Reply:
Good idea, I'll move it to the workflows dir.